README.md

```diff
@@ -87,15 +109,20 @@ $ git clone https://github.com/paritytech/parity-ethereum
 $ cd parity-ethereum
 
 # build in release mode
-$ cargo build --release --features final
+$ cargo build --release -p features final
```

Comment:
this is wrong,  `-p` is used to specify which crate to build, `--features` is something different.

Please revert the change